straxen/plugins/online_monitor.py

```diff
@@ -254,24 +254,130 @@ def area_width_hist(self, data):
         return hist.T
 
 
+@export
+@strax.takes_config(
+    strax.Option(
+        'drift_time_vs_R2_cut_string',
+        type=str,
+        default='(s2_index > -1) & (s1_index > -1)',
+        help='Selection (like selection_str) applied to data for '
+             '"drift_time_vs_R2", cuts should be separated using "&"'),
+    strax.Option(
+        'drift_time_vs_R2_bounds',
+        type=tuple, default=((0, 5e3), (0, 3e6)),
+        help='Boundaries of histogram of drift time vs R^2.'
+             'y_deges'),
+    strax.Option(
+        'online_monitor_nbins',

```

Comment:
perhaps make it a little more explicit that this is at event level, I should do the same at peak level.